src/VisualStudio/Core/Test/CodeModel/CSharp/FileCodeModelTests.vb

```diff
@@ -919,9 +921,55 @@ class $$C
                         Dim count = codeClass.Members.OfType(Of EnvDTE80.CodeFunction2)().Count()
                     End Sub)
             End Using
-
         End Sub
 
+        <WorkItem(1980, "https://github.com/dotnet/roslyn/issues/1980")>
+        <ConditionalFact(GetType(x86)), Trait(Traits.Feature, Traits.Features.CodeModel)>
+        Public Sub CreateUnknownElementForConversionOperator()
+            Dim oldCode =
+<Code>
+class D
+{
+    public static implicit operator D(double d)
+    {
+        return new D();
+    }
+}
+</Code>
+            Dim newCode =
+<Code>
+class D
+{
+}
+</Code>
+
+            Dim newDefinition =
+<Workspace>
+    <Project Language=<%= LanguageName %> CommonReferences="true">
+        <Document><%= newCode.Value %></Document>
+    </Project>
+</Workspace>
+
+            Using workspaceAndFileCodeModel = CreateCodeModelTestState(GetWorkspaceDefinition(oldCode))
+                Using workspace = TestWorkspaceFactory.CreateWorkspace(newDefinition, exportProvider:=VisualStudioTestExportProvider.ExportProvider)
```

Comment:
Names are a little confusing. Maybe "originalWorkspaceWithFileCodeModel" and "changedWorkspace" (to keep with the naming scheme used below)?